WTWSMS/history/characters/greek.txt

```diff
@@ -3802,6 +3827,18 @@
 		death=yes
 	}
 }
+13100018 = {
+    name="Callinicus" #PLRE vol.3 Callinicus 1
+	religion="nicene"
+	culture="greek"
+	dynasties=1310001
+	499.1.1={
+		birth=yes
+	}
+	548.1.1={
+		death=yes
+	}
+}
```

Comment:
This character should be a count, I think.

Reply:
PLRE says he was governor of Cilicia, and actually fits with the time his predecessor left office.

Reply:
Praeses of Ciliciae Secunda rather.

Reply:
Cilicia Prima and Cilicia Secuda were separate provinces with Cilicia Prima being the more important one. I think @Abian36 confused both these provinces, treating governors of both provinces as just governors of Cilicia.
For reference, see the setup in Cilicia in the pre-476 history branch (some commits ago, though, as it's currently temporarily broken after the merge from master branch).

Reply:
The problem is that the mod only has one Cilicia.
Of course I´ll check how you have managed to solve it.

Reply:
The problem could be solved by at least giving the governors of Cilicia Secunda counties in the area of the province and making them vassals of k_cilicia, held by governors of Cilicia Prima.
In my branch I created the d_cilicia_secunda title, but it is not necessary in master branch if the team doesn't want to change the setup.

Reply:
I´ve seen what you have done and I think it is optimous but still not 100% sure. We´ll discuss this in the evening.